### PR TITLE
[RFR]Update path to Checknode type

### PR DESCRIPTION
**Pull request by rlbabyuk**
Purpose or Intent
=================
Checknode, Unchecknode named tuple were added to widget. Fix tests

{{pytest:  cfme/tests/cloud/test_provisioning.py cfme/tests/configure/test_access_control.py cfme/tests/cloud_infra_common/test_tag_visibility.py -k 'test_tag_vis_vm or test_tag or test_group_crud_with_tag or test_cloud_provision_with_tag' --long-running}}

### CODE REVIEW

**Comment by dajoRH** [comment]
Lint report for commit 27dc320998459223d6436d22220ea429821e75e2:

`cfme/tests/cloud/test_provisioning.py`:
- :bangbang: Line 72:101: **E501** *line too long (128 > 100 characters)*

`cfme/tests/infrastructure/test_provisioning_dialog.py`:
- :bangbang: Line 238:101: **E501** *line too long (106 > 100 characters)*

Please, rectify these issues :smirk: .


**No commit flaws detected**.

*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit 49414aed415df531a17f712ad3b6442ebb4f911f:

`cfme/fixtures/tag.py`:
- :red_circle: Line 3:1: **N813** *camelcase imported as lowercase*

`cfme/tests/cloud/test_provisioning.py`:
- :red_circle: Line 10:1: **N813** *camelcase imported as lowercase*

`cfme/tests/infrastructure/test_provisioning_dialog.py`:
- :red_circle: Line 9:1: **N813** *camelcase imported as lowercase*

Please, rectify these issues :smirk: .


**No commit flaws detected**.

*CFME QE Bot*

**Comment by mshriver** [comment]
I think this duplicates https://github.com/ManageIQ/integration_tests/pull/6507/files, please review there!

**Comment by rlbabyuk** [comment]
Waiting for 6507 to be merged

File: cfme/tests/cloud/test_provisioning.py [hunk 1]
```diff
@@ -68,7 +69,8 @@ def testing_instance(request, setup_provider, provider, provisioning, vm_name, t
         parameter = request.param
         if parameter == 'tag':
             inst_args['purpose'] = {
-                'apply_tags': ('{} *'.format(tag.category.display_name), tag.display_name)
+                'apply_tags': Check_tree.CheckNode(
```
Comment (mshriver) [inline cfme/tests/cloud/test_provisioning.py]: :+1: Good catch, missed this one. Was particularly hard to find instances of these paths being filled!

**Comment by mshriver** [comment]
Network outage is causing PRT failures, but there are passing provisioning tests with tagging, and tests getting past the changes here without exception. Thanks for following up #6507 @rlbabyuk

**Pull request merged**